lms/djangoapps/course_api/blocks/tests/test_views.py

```diff
@@ -203,10 +203,14 @@ def test_block_counts_param(self):
             )
 
     def test_student_view_data_param(self):
-        response = self.verify_response(params={'student_view_data': ['video', 'chapter']})
+        response = self.verify_response(params={'student_view_data': ['video', 'chapter', 'discussion']})
         self.verify_response_block_dict(response)
         for block_data in response.data['blocks'].itervalues():
-            self.assert_in_iff('student_view_data', block_data, block_data['type'] == 'video')
+            self.assert_in_iff(
+                'student_view_data',
+                block_data,
+                block_data['type'] == 'video' or block_data['type'] == 'discussion'
```

Comment:
In that case, instead of repeating the list of blocks with `student_view_data` defined, can you create a global/local variable (e.g., `BLOCK_TYPES_WITH_STUDENT_VIEW_DATA`) that lists block types with `student_view_data` (`['video', 'discussion']`)?  And then you can replace this line with `block_data['type'] in BLOCK_TYPES_WITH_STUDENT_VIEW_DATA`.